examples/cluster/index.ts

```diff
@@ -66,7 +66,7 @@ const cluster4 = new eks.Cluster(`${projectName}-4`, {
 const coredns = new eks.Addon("coredns", {
   cluster: cluster3,
   addonName: "coredns",
-  addonVersion: "v1.11.1-eksbuild.9",
```

Comment:
We should rather dynamically fetch the correct version using https://www.pulumi.com/registry/packages/aws/api-docs/eks/getaddonversion/. That way we'll not have to keep updating this

You'll have to do the same for the `cluster-py` example.

Reply:
Closing in favor of https://github.com/pulumi/pulumi-eks/pull/1441 I think that's fixed there with dynamic fetching.